tests/test_apiclient.py

```diff
@@ -1300,7 +1300,8 @@ def test_find_audit_events_with_all_params(self, data_client, rmock):
             acknowledged='all',
             audit_date='2010-01-01',
             object_type='foo',
-            object_id=123)
+            object_id=123,
+            latest_first='True')
```

Comment:
This probably should be a boolean `True`, not a string. At least that's how I'd expect the method to be called.